Funcky.Async.Test/Extensions/AsyncEnumerableExtensions/InspectEmptyTest.cs

```diff
@@ -0,0 +1,37 @@
+#pragma warning disable SA1010 // StyleCop support for collection expressions is missing
```

Comment:
I think this is unneeded as there are no collection expressions in this file, right?

Reply:
removed in 00651d86a1d0d8e4988e4f26c4b3817ad682bd10